examples/settings.py

```diff
@@ -95,6 +98,23 @@ def make_cfg(settings):
         ortho_sensor_spec.sensor_subtype = habitat_sim.SensorSubType.ORTHOGRAPHIC
         sensor_specs.append(ortho_sensor_spec)
 
+    if settings["fisheye_sensor"]:
+        fisheye_sensor_spec = habitat_sim.FisheyeSensorDoubleSphereSpec()
+        fisheye_sensor_spec.uuid = "fisheye_sensor"
+        fisheye_sensor_spec.sensor_type = habitat_sim.SensorType.COLOR
+        fisheye_sensor_spec.sensor_subtype = (
+            habitat_sim.FisheyeSensorModelType.DOUBLESPHERE
+        )
+        fisheye_sensor_spec.resolution = [settings["height"], settings["width"]]
+        fisheye_sensor_spec.focal_length = (
+            np.ones(2) * min(settings["height"], settings["width"]) * 0.5
+        )
+        fisheye_sensor_spec.principle_point_offset = np.ones(2, dtype=np.int32) * (
+            min(settings["height"], settings["width"]) // 2
```

Comment:
I think the principle offset is not correct. 
(For Principal Point Offset, see here: http://ksimek.github.io/2013/08/13/intrinsic/)
This is what I did in the viewer:
https://github.com/facebookresearch/habitat-sim/blob/master/src/utils/viewer/viewer.cpp#L455-L457

Reply:
Also you need to set the 
    spec->xi = -0.18;
    spec->alpha = 0.59;
   somewhere.

Reply:
These are already the defaults in the sensorspec so no need to adjust the values though?

Reply:
https://github.com/facebookresearch/habitat-sim/blob/fc7baba4f208a7589a7af752ab686e39cccf28c4/src/esp/sensor/FisheyeSensor.h#L77

Reply:
Yeah, I would still encourage you to explicitly set them here. We are writing examples for the external users. So people know these are the parameters that they can tune and reset.

Reply:
Fixed it. It was a typo that wasn't caught due to the fact that we allow dynamic attrs. I thought for sure it would crash if that value wasn't set since it wasn't initialized.

@bigbike Should we make principal point set automatically from resolution like CubeMap size? I don't see many cases where you would want the principal point of center, right?